src/search/SearchMessagesScreen.js

```diff
@@ -37,17 +38,17 @@ class SearchMessagesScreen extends PureComponent<Props, State> {
    *  Performs a network request associated with a query. Does not
    *  update or access internal state (except `auth`).
    */
-  performQueryRaw = async (query: string): Promise<Message[]> => {
+  performQueryRaw = async (query: string) => {
```

Comment:
The type 'ServerApiResponseMessages' is not exported and can not be used outside of `api/getMessages.js` so I have removed the flow type here.